src/index-rollup-stable.js

```diff
@@ -57,6 +47,15 @@ module.exports = (options = {}) => {
      * @return {void}
      */
     renderChunk(code, chunk, outputOptions = {}) {
+      plugin.scanDependencies(
+          _.chain(chunk.modules)
+              .toPairs()
+              .reject((mod) => mod[1].isAsset)
+              .filter((mod) => mod[1].renderedLength > 0)
```

Comment:
I told you that renderedLength is 0 if everything is treeshaken but would be nice to verify this is actually the case😬 Otherwise this looks good to me!

Reply:
I can confirm `renderedLength === 0` works as long as the file does not have a leading comment, which is always retained. But I guess this is something we can fix on Rollup's side, so I do not see any issues here as well.

Reply:
Thanks @lukastaegert, sorry for the delay, but I had a lot of work ;)